packages/larva/scripts/link-all.js

```diff
@@ -0,0 +1,13 @@
+/**
+ * This command can be used from within any Larva project to link the Larva
+ * packages (from the monorepo) for local development.
+ *
+ * @see https://github.com/penske-media-corp/pmc-larva/blob/master/README.md#linking-larva-for-local-development
+ */
+
+var shell = require( 'shelljs' );
+
+shell.exec( 'npm link @penskemediacorp/larva' );
+shell.exec( 'npm link @penskemediacorp/larva-patterns' );
+shell.exec( 'npm link @penskemediacorp/larva-css' );
+shell.exec( 'npm link @penskemediacorp/larva-js' );
```

Comment:
Can we also add the remaining packages?

npm link @penskemediacorp/larva-scss
npm link @penskemediacorp/larva-tokens
npm link @penskemediacorp/twig-to-php-parser

Reply:
@laras126, I only accounted for the packages currently listed out in the Confluence docs - what do you think about Sagar's point? Any context I should be aware of before adding them?

Reply:
I agree to add them! They are not commonly linked so I think not included in confluence.